src/constants/r2Prefixes.ts

```diff
@@ -3,11 +3,23 @@
 //  later on.
 // (e.g. url path `/dist` points to R2 path `nodejs/release`)
 // See https://raw.githubusercontent.com/nodejs/build/main/ansible/www-standalone/resources/config/nodejs.org
+import map from './redirectLinks.json';
```

Comment:
Does this file exists? Or is it built at some point?

Reply:
auto-generated: https://github.com/nodejs/release-cloudflare-worker/pull/34